package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@altis/cornerstone-tools",
-  "version": "1.1.5",
+  "version": "1.1.6",
```

Comment:
This should be `1.1.7` I believe

Reply:
c3cc50154230ac21ba11c9f00428ad2d0a1b4897